conans/cli/commands/inspect.py

```diff
@@ -0,0 +1,31 @@
+import json
+
+from conans.cli.command import conan_command, COMMAND_GROUPS
+from conans.cli.output import ConanOutput
+from conans.errors import ConanException
+
+
+def _inspect_json_formatter(data):
+    return json.dumps(data, indent=4)
+
+
+@conan_command(group=COMMAND_GROUPS['creator'], formatters={"json": _inspect_json_formatter})
```

Comment:
Maybe the inspect is useful as consumer, to know for example which options does a recipe have?